src/libraries/Microsoft.PowerFx.Core/Lexer/TexlLexer.cs

```diff
@@ -246,6 +246,11 @@ public static TexlLexer GetLocalizedInstance(CultureInfo culture)
         public static IReadOnlyList<string> GetKeywords()
         {
             return _keywords.Keys.ToList();
+        }
+
+        public static IReadOnlyList<string> GetReservedKeywords()
```

Comment:
Ienumerable? then we can avoid ToList call.

Reply:
This has been removed since we adopted the inline approach in the new test cases.